modules/entity_version_history/src/HistoryTabSettingsInterface.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\entity_version_history;
+
+use Drupal\Core\Config\Entity\ConfigEntityInterface;
+
+/**
+ * Provides an interface defining history tabs settings for content entities.
+ */
+interface HistoryTabSettingsInterface extends ConfigEntityInterface {
+
+  /**
+   * Gets the entity type ID this config applies to.
+   *
+   * @return string
+   *   Returns the target entity type id.
+   */
+  public function getTargetEntityTypeId(): string;
```

Comment:
Aren't you missing a setter for this? The others have it. 

Also please group the methods together, getter and setter one after the otherr.